source/localizable/routing/asynchronous-routing.md

```diff
@@ -26,7 +26,7 @@ function fulfill(answer) {
 }
 
 function reject(reason) {
-  console.log(`Couldn't get the answer! Reason: ${reason}`);
+  console.log('Couldnot get the answer! Reason: ${reason}');
```

Comment:
"Could not" should be 2 words. Also, this would need to still be a template string. Single quotes will display "${reason}" instead of the value of `reason`